tests/aktualizr.supp

```diff
@@ -55,6 +55,15 @@
    fun:_ZN5boost10filesystem11unique_pathERKNS0_4pathE
    ...
 }
+{
+   boost 1.66.0 unique_path
+   Memcheck:Addr16
+   ...
+   fun:_ZN5boost10filesystem11path_traits7convertEPKwS3_RNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEERKSt7codecvtIwc11__mbstate_tE
+   fun:_ZN5boost10filesystem6detail11unique_pathERKNS0_4pathEPNS_6system10error_codeE
+   fun:_ZN5boost10filesystem11unique_pathERKNS0_4pathE
+   ...
+}
```

Comment:
Are you sure this leak isn't real?  Do you have a link to an upstream bug or similar?

Reply:
This looks related to the errors I added there: https://github.com/advancedtelematic/aktualizr/pull/488

Maybe the rules can be merged.

Unfortunately, I didn't find any other mention upstream...

Reply:
I got the test failed on my setup. And I think it is also related to #488 (boost::unique_path), except for valgrind category: in my case it should be under 'Memcheck:Addr16'.
I'll check, may be it is possible to merge the categories only.

Reply:
There is no way to combine categories - left it as is.